pkgs/by-name/um/umu-launcher-unwrapped/package.nix

```diff
@@ -1,51 +1,67 @@
 {
-  python3Packages,
-  fetchFromGitHub,
-  lib,
   bash,
+  cargo,
+  fetchFromGitHub,
   hatch,
+  lib,
+  python3Packages,
+  rustPlatform,
   scdoc,
-  replaceVars,
-  fetchpatch2,
+  withTruststore ? true,
+  withDeltaUpdates ? true,
 }:
 python3Packages.buildPythonPackage rec {
   pname = "umu-launcher-unwrapped";
-  version = "1.1.4";
+  version = "1.2.3";
 
   src = fetchFromGitHub {
     owner = "Open-Wine-Components";
     repo = "umu-launcher";
     tag = version;
-    hash = "sha256-TOsVK6o2V8D7CLzVOkLs8AClrZmlVQTfeii32ZIQCu4=";
+    hash = "sha256-7Q9G8eNkEB/ksBix6AcAqs8LyQ6AHx4hR5h003yo7GQ=";
   };
 
-  # Both patches can be safely removed with the next release
-  patches = [
-    # Patch to avoid running `git describe`
-    # Fixed by https://github.com/Open-Wine-Components/umu-launcher/pull/289 upstream
-    (replaceVars ./no-umu-version-json.patch { inherit version; })
-    # Patch to use PREFIX in the installer call
-    (fetchpatch2 {
-      url = "https://github.com/Open-Wine-Components/umu-launcher/commit/602a2f84a05a63f7b1b1c4d8ca85d99fdaec2cd2.diff";
-      hash = "sha256-BMinTXr926V3HlzHHabxHKvy8quEvxsZKu1hoTGQT00=";
-    })
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit src;
+    hash = "sha256-nU4xZn9NPd7NgexiaNYLdo4BCbH98duZ07XYeUiceP0=";
+  };
+
+  nativeCheckInputs = [
+    python3Packages.pytestCheckHook
```

Comment:
Some of upstream's CI mentions `unittest`, however adding `unittestCheckPhase` didn't seem to run any additional tests:

```
Executing unittestCheckPhase
unittest flags: ''

----------------------------------------------------------------------
Ran 0 tests in 0.000s

NO TESTS RAN
```

The docs also mention:

> `pytest` is compatible with `unittest`, so in most cases you can use `pytestCheckHook` instead [of `unittestCheckPhase`].